crates/jwk/src/algorithm.rs

```diff
@@ -26,7 +26,7 @@ macro_rules! algorithms {
             ///
             /// Per the specs it should only be `none` but `None` is kept for backwards
```

Comment:
It would be nice to have a test to check that it serializes as `none` and that it is able to deserialize both.

Reply:
Good call, it was serializing to `none`.